{"uuid":"dd216f41-1b1b-4183-bdc2-78fe9a7f02f7","name":"Scenario Outline: The user getting error message with invalid python code for Insertion page","children":["fde04780-579e-470d-9b9f-8d6f4a355b21"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"bd5afeac-d408-4d97-acc5-93f5112c9834-attachment","type":"image/png"},{"name":"Myscreenshot","source":"de50a3f8-e608-4b81-b92e-6e844307c49c-attachment"}],"parameters":[],"start":1696885868048,"stop":1696885868240},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885868240,"stop":1696885868240},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885868241,"stop":1696885868241},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885868242,"stop":1696885868242}],"start":1696885868045,"stop":1696885868243}